babycenter/tests/test_pagination.py

```diff
@@ -0,0 +1,104 @@
+        self.assertContains(response, '<a href="?p=2">2</a>')
+        self.assertContains(response, '<a href="?p=4">4</a>')
+        self.assertContains(response, '<a href="?p=5">5</a>')
+        self.assertContains(response, '...')
+        self.assertContains(response, '<a href="?p=15">15</a>')
+        self.assertContains(response, '<a href="?p=16">16</a>')
+        self.assertContains(response, 'Next')
+
+    def test_fifth_page_pagination(self):
+        response = self.client.get('/sections/english-section/?p=5')
+        self.assertContains(response, 'Prev')
+        self.assertContains(response, '<a href="?p=1">1</a>')
+        self.assertContains(response, '<a href="?p=2">2</a>')
+        self.assertContains(response, '...')
+        self.assertContains(response, '<a href="?p=3">3</a>')
+        self.assertContains(response, '<a href="?p=4">4</a>')
+        self.assertContains(response, '<a href="?p=6">6</a>')
+        self.assertContains(response, '<a href="?p=7">7</a>')
+        self.assertContains(response, '...')
+        self.assertContains(response, '<a href="?p=15">15</a>')
+        self.assertContains(response, '<a href="?p=16">16</a>')
+        self.assertContains(response, 'Next')
+
+    def test_fourteenth_page_pagination(self):
+        response = self.client.get('/sections/english-section/?p=14')
+        self.assertContains(response, 'Prev')
+        self.assertContains(response, '<a href="?p=1">1</a>')
+        self.assertContains(response, '<a href="?p=2">2</a>')
+        self.assertContains(response, '...')
+        self.assertContains(response, '<a href="?p=12">12</a>')
+        self.assertContains(response, '<a href="?p=13">13</a>')
+        self.assertContains(response, '<a href="?p=15">15</a>')
+        self.assertContains(response, '<a href="?p=16">16</a>')
+        self.assertContains(response, 'Next')
+
+    def test_fifteenth_page_pagination(self):
+        response = self.client.get('/sections/english-section/?p=15')
+        self.assertContains(response, 'Prev')
+        self.assertContains(response, '<a href="?p=1">1</a>')
+        self.assertContains(response, '<a href="?p=2">2</a>')
+        self.assertContains(response, '...')
+        self.assertContains(response, '<a href="?p=13">13</a>')
+        self.assertContains(response, '<a href="?p=14">14</a>')
+        self.assertContains(response, '<a href="?p=16">16</a>')
+        self.assertContains(response, 'Next')
+
+    def test_sixteenth_page_pagination(self):
+        response = self.client.get('/sections/english-section/?p=16')
+        self.assertContains(response, 'Prev')
+        self.assertContains(response, '<a href="?p=1">1</a>')
+        self.assertContains(response, '<a href="?p=2">2</a>')
+        self.assertContains(response, '...')
+        self.assertContains(response, '<a href="?p=14">14</a>')
+        self.assertContains(response, '<a href="?p=15">15</a>')
+        self.assertNotContains(response, 'Next')
```

Comment:
Why do we only show up to 16 pages even though there are more than 16 (200 in this case)?